routes/userRoutes.js

```diff
@@ -15,6 +16,10 @@ router.delete('/:id', requireLogin, async (req, res) => {
 		}
 		const id = mongoose.Types.ObjectId(req.params.id);
 
+		// if (req?.tokenClaims?.id !== id && req?.tokenClaims?.role !== 'admin') {
+		// 	return res.status(401).send({ error: errorCodes['E0002'] });
+		// }
+
```

Comment:
This commented code checks if the request is made by the owning user or an admin. If this is commented out any user can delete any other user.
This needs to be fixed!